plugins/modules/source_control/gitlab/gitlab_group.py

```diff
@@ -258,6 +280,9 @@ def main():
         state=dict(type='str', default="present", choices=["absent", "present"]),
         parent=dict(type='str'),
         visibility=dict(type='str', default="private", choices=["internal", "private", "public"]),
+        project_creation_level=dict(type='str', default="developer", choices=["developer", "maintainer", "noone"]),
```

Comment:
Please use `'` for all strings (and not `"` for some). This is already inconsequent for existing options, there's no need to keep doing that for new ones :)